### DISCUSSION TITLE
Admin page available when accessing Rdio-scanner over the internet. How can I disable that / make sure I am secure?

### DISCUSSION THREAD

**Discussion by RuziiRich** [chuot/rdio-scanner#403] [Q&A]
so I have exposed rdio-scanner to the internet, and I noticed that my admin page is available too when searching **mydomain.com/admin**

I set a super strong password, but I would like to not have this admin page available at all over the internet. Is this possible? is my concern valid here? 

If hiding this page isnt possible, is there a way I can secure this page to protect again brute force attacks for example. My password should not brute forced easily as its very long, but still want to make sure I am secure.

**Comment by Thinline-Dynamic-Solutions** [comment]
I don’t believe so. Just use a good strong password.

*Michael Chambers - Ohio Radio Scanner Network*
Founder/Manager/Support
***@***.**** ***@***.***>

**Comment by adorsett** [comment]
Having an ability to set an IP ACL on the /admin endpoint would be a good security enhancement.

**Reply by RuziiRich** [reply]
That would be a good solution! I don't like having the admin page publicity accessible even if I do have a strong password to protect it.

**Comment by kd4e** [comment]
If it's only you accessing it you might consider not directly exposing it to the Internet but, rather, accessing it as a 'terminal' via RustDesk (or similar).

**Comment by infecticide** [comment]
You could setup an Apache or Nginx reverse proxy and disallow access to the /admin subfolder.

**Reply by RuziiRich** [reply]
That's a great point, thank you.

**Reply by someengineer1** [reply]
I filter out /api also just in case, should not be any reason for an external user to access this (assuming you aren't uploading your calls from outside).  Adding cloudflare as a front end also adds a lot of security and flexibility too, can even offload the SSL to them and just run port 80 to rdio, less load on your box and no security issue since it is going inside of a VPN tunnel and there is nothing sensitive being sent there anyway (unless you're using access codes).

**Comment by someengineer1** [comment]
I'm using a combination of cloudflare filtering (front ending rdio scanner with cloudflare zero trust tunnel which is free, and lets them handle all the attacks and you can filter out countries etc, all for free) plus Caddy reverse proxy running on the local machine.  Pick the reverse proxy of your choice, I like caddy, nginx and others are good too.

With both (for dual layers of protection) I filter out /admin* and /api* from being accessed externally, along with obviously having a strong password on the admin interface just in case.  Does anyone know if there are others that should be blacklisted?  Can't use a whitelist as rdio actually uses quite a few hidden URLs to function.
